flamingpy/examples/surface_code.py

```diff
@@ -15,7 +15,10 @@
 import matplotlib.pyplot as plt
 import numpy as np
 
-from flamingpy.codes.surface_code import SurfaceCode, alternating_polarity
+from flamingpy.codes.surface_code import (
+    SurfaceCode,
+    alternating_polarity,
+)  # pylint: disable=unused-import
```

Comment:
Currently, we have decided to add pylint ignores on the top only -- while there is a point in adding them in place, we prefer consistency and manageability here.